package/yast2-iplb.spec

```
@@ -63,5 +63,6 @@ used on high availability (HA) clusters.
 %{yast_scrconfdir}/*.scr
 %doc %{yast_docdir}
 %license COPYING
+%{_datadir}/icons/*
```

Please use `%{yast_icondir}`.

```suggestion
%{yast_icondir}
```